src/System.Private.CoreLib/shared/System/BitOps.cs

```diff
@@ -203,6 +204,50 @@ public static int RotateLeft(int value, int offset)
 
         #endregion
 
+        #region LogBase2
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static uint LogBase2(uint value)
```

Comment:
This should probably be called `Log2`, it's also one that might need to go onto `Math`, but that should require further API discussion.

Reply:
Renamed. Made a note of moving it to `Math`

Reply:
`ILogB` is an explicit IEEE operation that operates on the `base` power of the input. So the IEEE `binary` types (`binary16`, `binary32` aka `System.Single`, `binary64` aka `System.Double`, etc) are effectively `ILog2`. However, the IEEE `decimal` types (`decimal32`, `decimal64`, etc) would be effectively `ILog10`. We also have to prefix with `I` because it takes a float/double and returns an integer and you can't overload based on return type (it also matches the IEEE name).

The correct name for the method that take an `int` and return an `int` is just `Log2`

Reply:
My concern is that `double Math.Log2(double)` already exists, and so I believe adding `uint Math.Log2(uint)` would be a breaking change. I can name it that here, but I think not in `Math`

Reply:
> I can name it Log2 here, but I think not in Math

A right, implicit conversions and what not (same reason we had to create `MathF`).